source/extensions/filters/http/cors/cors_filter.cc

```diff
@@ -200,11 +205,37 @@ bool CorsFilter::allowCredentials() {
   return false;
 }
 
+absl::optional<std::string> CorsFilter::runtimeKey(const std::string& key) {
+  for (const auto policy : policies_) {
+    if (!policy) {
+      continue;
+    }
+    if (!policy->runtimeKey().empty()) {
+      return fmt::format("cors.{}.{}", policy->runtimeKey(), key);
+    }
+  }
+  return EMPTY_STRING;
```

Comment:
@lizan this should actually return `absl::nullopt`.

Reply:
Yeah that works too, but for consistency with other CorsPolicy, slightly prefer using an empty string.